src/client/config/local.config.json

```diff
@@ -1,4 +1,5 @@
 {
   "overwriteServerEndpoint": true,
-  "serverEndPointUrl": "localhost"
-}
\ No newline at end of file
+  "serverEndPointUrl": "localhost",
+  "serverPort": 8000
```

Comment:
We will need to find a better naming convention in the future as we have multiple way to dev in local (installed on machine / docker / minikube / ...)

But this is an issue we did have in the past.

To me it's ok for this.